File: Scripts/newTask/call-an-object-method/swift/call-an-object-method.swift
class CallAnObjectMethod{

  // stored property
  var variable : Int

  /**
  * The constructor
  */
  init() {
    self.variable = 42
  }

  /**
  * A method
  */
  func someMethod(a: Int) {
    self.variable = a
  }
}

let value = CallAnObjectMethod()
for i in 0...1000000000 {
	value.someMethod(a: 1)
}

File: Scripts/newTask/call-an-object-method/php/call-an-object-method.php
<?php

class CallAnObjectMethod {
	public static $classVar;
    	function __construct() {
    	}
   
	function someMethod($a) {
        	self::$classVar = $a;
    	}
}

	$myObj = new CallAnObjectMethod();
	for ( $i = 0; $i < 1000000000; $i++) {
		$myObj->someMethod(1);
	}
?>
